int32 value
string name
---
int32 error_code
string error_description